index.html

```diff
@@ -2356,9 +2401,39 @@
       iframe.addEventListener('destroy', () => {
         screenQuad.material.uniforms.numTextures.value = 1;
       });
-      container.appendChild(iframe);
-
-      iframes.push(iframe);
+        // function to test the URL on submit for a 404
+        function checkURL() {
+            let request;
+            if(window.XMLHttpRequest)
+                request = new XMLHttpRequest();
+            else
+                request = new ActiveXObject("Microsoft.XMLHTTP");
+                try{
+                    console.log("Loading");
+                    request.open('GET', u, false);
```

Comment:
This looks like a synchronous `open`.

That means the user's face literally lock in place (or you lose world rendering entirely) while this request goes through/fails. In addition, I think this is deprecated behavior per the spec because of nasty reasons like this.

Basically, we can't use this technique.

https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/open